src/view/global.styl

```diff
@@ -14,12 +14,17 @@ governing permissions and limitations under the License.
 
 // Simplifies centering things vertically in the middle of the view (like loading errors) or
 // creating columns that are the full height of the view and separately scrollable.
-html,
+html {
+  height: 100%
+}
+
+html
 body,
 #root,
 .react-spectrum-provider {
-  box-sizing: border-box;
-  height: 100%;
+  display: flex;
+  flex-direction: column;
+  flex: 1
 }
```

Comment:
Previously the body would never grow larger than the height of the iframe. This resulted in no padding being displayed at the bottom of some views when scrolled to the bottom:

![image](https://user-images.githubusercontent.com/210820/117729982-e0daf480-b1a8-11eb-9768-389adb15e77f.png)


My changes fix that.